packages/block-library/src/buttons/style.scss

```diff
@@ -1,7 +1,14 @@
 .wp-block-buttons .wp-block-button {
 	display: inline-block;
-	margin: $grid-size-small;
+	margin-right: $grid-unit-10;
```

Comment:
During previous cherry-picks (e.g: during 5.4 RC) we opted for not include the new grid system and just change the variable to their equivalents e.g: $grid-unit-10 would be $grid-size in the "old system". But in retrospective probably including the grid system from the start would have been a safer and easier choice, so I don't have anything against including the new grid variables (I regret not having done it sooner). cc: @youknowriad in case you think including the grid system may have an undesirable side effect in future cherry-picks.

Reply:
I think it's fine to include it, it's harmless